proposals/first-class-span-types.md

```diff
@@ -445,25 +449,6 @@ However, that would mean users could get different behavior after updating the t
 On the other hand, the new Span APIs would still be ambiguous from VB unless we implement the betterness rule there as well.
 So it might be easier if API authors solve this themselves via the `OverloadResolutionPriorityAttribute`.
 
-### Delegate extension receiver break
```

Comment:
Rather than removing this section, please mark it answered and put the answer as a subheading below, like the other answered questions above.

Reply:
Previously it was suggested to remove the resolved question if its answer is somewhere else in the speclet: https://github.com/dotnet/csharplang/pull/8241#discussion_r1653603180. I find that reasonable

Reply:
Let's please keep the question, as that will make it make more sense when people go from the notes to the current version of the speclet.